<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramPreferencesDialog</class>
 <widget class="QDialog" name="ProgramPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>311</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences and Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="programSettingsTab">
      <attribute name="title">
       <string>Program Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>im4j Search Path:</string>
           </property>
           <property name="buddy">
            <cstring>im4jLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="im4jLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="exiftoolPathEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="exiftoolBrowseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="dcrawPathEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="imageMagickPathEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="dcrawBrowseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="imageMagickBrowseButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="exiftoolOverrideCheckBox">
           <property name="text">
            <string>Override exiftool:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="dcrawOverrideCheckBox">
           <property name="text">
            <string>Override dcraw:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="imageMagickOverrideCheckBox">
           <property name="text">
            <string>Override ImageMagick:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="useGMCheckBox">
           <property name="text">
            <string>Try to use GraphicsMagick instead of ImageMagick</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="collectionSettingsTab">
      <attribute name="title">
       <string>Collection Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Collection Name:</string>
           </property>
           <property name="buddy">
            <cstring>collectionNameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="collectionNameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Owner Name:</string>
           </property>
           <property name="buddy">
            <cstring>ownerNameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ownerNameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>110</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>im4jLineEdit</tabstop>
  <tabstop>exiftoolOverrideCheckBox</tabstop>
  <tabstop>exiftoolPathEdit</tabstop>
  <tabstop>exiftoolBrowseButton</tabstop>
  <tabstop>dcrawOverrideCheckBox</tabstop>
  <tabstop>dcrawPathEdit</tabstop>
  <tabstop>dcrawBrowseButton</tabstop>
  <tabstop>imageMagickOverrideCheckBox</tabstop>
  <tabstop>imageMagickPathEdit</tabstop>
  <tabstop>imageMagickBrowseButton</tabstop>
  <tabstop>useGMCheckBox</tabstop>
  <tabstop>collectionNameLineEdit</tabstop>
  <tabstop>ownerNameLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exiftoolOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>exiftoolPathEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dcrawOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dcrawPathEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageMagickOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageMagickPathEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exiftoolOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>exiftoolBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dcrawOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dcrawBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageMagickOverrideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageMagickBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
